README.md

```diff
@@ -97,7 +97,6 @@ and [Git server](examples/git) in the [examples](examples) folder.
 * [Wishlist](https://github.com/charmbracelet/wishlist)
 * [SSHWordle](https://github.com/davidcroda/sshwordle)
 * [clidle](https://github.com/ajeetdsouza/clidle)
-* [ssh-warm-welcome](https://git.vvvvvvaria.org/decentral1se/ssh-warm-welcome)
```

Comment:
new url seems to be https://git.coopcloud.tech/decentral1se/ssh-warm-welcome

Reply:
Good spot, updated in https://github.com/charmbracelet/wish/pull/172/commits/2d4d11c5e9473306b7dc462bfebcb54483440473. 👍🏼